mangaki/mangaki/tests/test_anidb.py

```diff
@@ -47,5 +52,19 @@ def test_anidb_get(self):
             content_type='application/xml'
         )
         anime = self.anidb.get_or_update_work(11606)
-        self.assertNotEqual(anime.title, '')
-        self.assertNotEqual(anime.worktitle_set.count(), 0)
+
+        self.assertEqual(anime.title, 'Sangatsu no Lion')
+        self.assertEqual(anime.nb_episodes, 22)
+        self.assertEqual(anime.studio.title, 'Shaft')
+
+        author_ids = [q.artist_id for q in Staff.objects.filter(work_id=anime.pk, role_id=3)]
```

Comment:
This could be rewritten in an idiomatic Django as: `Staff.objects.filter(work_id=anime.pk, role_id=3).values_list('artist_id', flat=True)` (which will return a list of IDs).

Also, rather than to rely on numeric value such as ID, you could filter with `role__slug='author'`.

[About the `field__something`, it's called a lookup.](https://docs.djangoproject.com/fr/1.11/ref/models/querysets/#field-lookups)
You can either use a field of the foreign key considered, `role__slug` or `role__name` for example.
Or you could use integrated lookups to have different results `role__slug__contains='a'`.